src/functionalTest/java/uk/gov/hmcts/dts/fact/AdminCourtsEndpointTest.java

```diff
@@ -19,6 +19,8 @@
 @ExtendWith({SpringExtension.class})
 @SpringBootTest(classes = {OAuthClient.class})
 public class AdminCourtsEndpointTest {
+
+    private static final String SLUG = "birmingham_civil_and_family_justice_centre";
```

Comment:
birmingham-civil-and-family-justice-centre

Reply:
I found the slug already setup so removed this one